Okta/OktaOidc/Internal/OktaOidcRestApi.swift

```diff
@@ -102,11 +102,18 @@ extension OktaOidcHttpApiProtocol {
 }
 
 class OktaOidcRestApi: OktaOidcHttpApiProtocol {
+    var delegate: OktaOidcHTTPProtocol?
```

Comment:
Would go for naming of `OktaOidcRestApiDelegate` for the case of a delegate protocol

Reply:
Yes, I second that, it should indicate that this delegate allows the URlrequest to be modified before they are sent. Maybe  NetworkRequestCustomizationDelegate ??